cardano-streaming/cardano-streaming.cabal

```diff
@@ -0,0 +1,97 @@
+cabal-version: 3.0
+name:          cardano-streaming
+version:       1.2.0.0
+author:        Andrea Bedini
+maintainer:    andrea.bedini@iohk.io
+
+-- We don't care about ghc810, and ghc910 ships with base ==4.20.0, 
+-- while ouroboros-consensus (which we depend on) binds base <4.20.0. 
+-- TODO remove this once ouroboros-consensus supports ghc910
+common ghc-version-support
+  if ((impl(ghc >9.8) || impl(ghc <9.6)) || os(windows))
+    buildable: False
+
+common lang
+  default-language:   Haskell2010
+  default-extensions:
+    DeriveFoldable
+    DeriveFunctor
+    DeriveGeneric
+    DeriveLift
+    DeriveTraversable
+    ExplicitForAll
+    GeneralizedNewtypeDeriving
+    ImportQualifiedPost
+    LambdaCase
+    NamedFieldPuns
+    ScopedTypeVariables
+    StandaloneDeriving
+
+  ghc-options:
+    -Wall -Widentities -Wincomplete-record-updates
+    -Wincomplete-uni-patterns -Wmissing-import-lists
+    -Wnoncanonical-monad-instances -Wredundant-constraints
+    -Wunused-packages
+
+library
+  import:          lang, ghc-version-support
+  hs-source-dirs:  src
+  exposed-modules:
+    Cardano.Streaming
+    Cardano.Streaming.Callbacks
+    Cardano.Streaming.Helpers
+
+  --------------------------
+  -- Other IOG dependencies
+  --------------------------
+  build-depends:
+    , cardano-api
+    , ouroboros-network-protocols
+
+  ------------------------
+  -- Non-IOG dependencies
+  ------------------------
+  build-depends:
+    , async
+    , base                         >=4.9 && <5
+    , cardano-api
```

Comment:
This creates a circular dependency, unfortunately, as `cardano-api` depends on the `plutus`.